lib/katello/tasks/upgrade_check.rake

```diff
@@ -13,6 +13,7 @@ namespace :katello do
     task_count = ::ForemanTasks::Task.active.where("label != '#{CP_LISTEN_ACTION}'").count
     task_status = task_count > 0 ? fail : success
     puts "Checking for running tasks..."
-    puts "[#{task_status}] - There are #{task_count} active tasks.\n\n"
+    puts "[#{task_status}] - There are #{task_count} active tasks. "
+    puts "         Please wait for these to close or cancel them from the Monitor tab.\n\n"
```

Comment:
Maybe use finish or complete instead of close? Since 'close' is never shown in the UI, this may confuse users.